app/controllers/spree/admin/payments_controller.rb

```diff
@@ -35,15 +35,16 @@ def create
             return
           end
 
-          authorize_stripe_sca_payment
-
           if @order.completed?
-            @payment.process!
+            authorize_stripe_sca_payment
+            @payment.process_offline!
```

Comment:
why does it need to be processed offline now? could you explain it a bit more? is it to defend against a failed authorization step?

Reply:
This change wasn't really to defend against anything, but I think it is the correct thing to do here. We're taking a payment in the back office, so the customer isn't "online" to complete the payment. I suspect we didn't notice this before because we always went through the other branch of this conditional.

Reply:
good catch :clap: